frontend/app/controllers/Event.scala

```diff
@@ -218,7 +226,14 @@ trait Event extends Controller with ActivityTracking {
       } yield {
         event.service.getOrder(oid).map { order =>
           trackConversionToThankyou(request, event, Some(order), Some(request.member))
-          Ok(views.html.event.thankyou(event, order)).discardingCookies(DiscardingCookie(eventCookie(event)))
+
+          request.session.get(isComplimentaryKey) foreach ( _ =>
```

Comment:
:-1:  This looks wrong to me - it looks like you're assuming that because we showed complimentary tickets to the user (`session.get(isComplimentaryKey)`), the user chose to use them? I believe we should instead be using the Attendee information from the EventBrite APi data for the Order https://www.eventbrite.com/developer/v3/formats/attendee/#ebapi-std:format-attendee - look at the ticket classes consumed to see if a compilmentary ticket was used.